### ISSUE TITLE
Dismiss "Disable developer mode extensions"

### ISSUE THREAD

**Issue by Javran** [KC3Kai/KC3Kai#1884]
This issue serves as a project announcement so feel free to close this after several days.
But any further discussions or alternative solutions are always welcome to be put in this issue.

You might suffer from the annoying "Disable developer mode extensions" popup when you are working with dev version of KC3Kai, here I wrote a little project to temporarily solve the issue for linux users:

https://github.com/Javran/wedoevil

Basically what it does is just to detect the dialog, click "Cancel" and quit itself. but that's already better than having to do this by self.

There should have been many tools in Windows that does exactly the same, but I'll consider to support windows if there is a need.

**Comment by LostLogia4** [comment]
Even my locally saved extension directory pops this message up, so this is definitely not a dev-only problem.

**Comment by Javran** [comment]
@LostLogia4 yep, basically chrome devs consider everything is evil except whattheheckever installed from their web store.

**Comment by gensuihime** [comment]
Several days = passed
